banananation/src/main/java/gov/nasa/jpl/aerie/banananation/package-info.java

```diff
@@ -1,7 +1,6 @@
 @MissionModel(model = Mission.class)
```

Comment:
*Adding a comment here to group this discussion under a thread.*

We have 4 distinct styles of default parameters within activities and now configurations:
- All default arguments provided within `@Parameter` annotations
- All default arguments provided within `@Template` static method
- Some arguments provided within `@WithDefaults` static class
- No default arguments provided

Since only one configuration class can be defined for a mission model (`@WithConfiguration`) it may be nice to have a separate example Gradle subproject for each configuration style. Currently we have examples (Foo and Banananation respectively) for the first 2 styles listed above. I could create 2 new subjects that demonstrate the other 2 styles. Any thoughts here?

Reply:
Additionally, I would love to clean things up by creating a new top-level directory called `examples/` which contains Foo, Banananation, and any other example Gradle subprojects. This is a more standard directory structure and simply requires changing `settings.gradle`'s paths to these example projects. This would also encourage us to write more examples covering more mission model config. use cases without cluttering the top-most repo. directory.

Reply:
> Additionally, I would love to clean things up by creating a new top-level directory called `examples/` which contains Foo, Banananation, and any other example Gradle subprojects. This is a more standard directory structure and simply requires changing `settings.gradle`'s paths to these example projects. This would also encourage us to write more examples covering more mission model config. use cases without cluttering the top-most repo. directory.

Ooh, I like that.

Reply:
fa2ce2e7f4f297bc65a337ca24d291282719b042 moves these to `examples/`. I'll add 2 more minimal subprojects to that directory in another commit on this PR.

Reply:
`examples/config-with-defaults` and `examples/config-without-defaults` has been added as of latest push. Both subprojects exist to demonstrate mission model configuration with and without defaults "baked in". Tested with two `ConfigurationTest` classes.